static/js/zamboni/collections.js

```diff
@@ -660,8 +660,8 @@ $(document).ready(function () {
                     $widget.setWidth(410);
                     $widget.setPos(ct);
                     $("#id_name").focus();
-                    $widget.delegate('#collections-new-cancel', 'click', loadList)
-                           .delegate('#add-to-collection form', 'submit', handleSubmit);
+                    $('#collections-new-cancel').on('click', loadList);
```

Comment:
I saw that, but because we aren't chaining the same element (and are selecting unique elements -- IDs) I found this usage of the API cleaner.